pkg/assets/handlers.go

```diff
@@ -52,6 +52,15 @@ func GzipHandler(h http.Handler) http.Handler {
 	})
 }
 
+func BrowserSecurityHandler(h http.Handler) http.Handler {
```

Comment:
I have a small preference for `SecurityHeadersHandler`, which makes it more obvious what the handler is adding